examples/non_cartesian_reconstruction_stack_3d.py

```diff
@@ -60,15 +74,20 @@
 kspace_obs = fourier_op.op(image.data)
 
 # Gridded solution
-grid_space = [np.linspace(-0.5, 0.5, num=image.shape[i])
-              for i in range(len(image.shape) - 1)]
+grid_space = [np.linspace(-0.5, 0.5, num=img_shape)
+              for img_shape in image.shape[:-1]]
 grid = np.meshgrid(*tuple(grid_space))
-kspace_plane_loc, z_sample_loc, sort_pos = get_stacks_fourier(kspace_loc)
-grid_soln = gridded_inverse_fourier_transform_stack(kspace_plane_loc,
-                                                    z_sample_loc,
-                                                    kspace_obs,
-                                                    tuple(grid),
-                                                    'linear')
+kspace_plane_loc, z_sample_loc, sort_pos, idx_mask_z = get_stacks_fourier(
+    kspace_loc,
+    image.shape)
+grid_soln = gridded_inverse_fourier_transform_stack(
```

Comment:
Ahh, this is bad, we would want the stacking done by the function itself. Can we have an issue for this?
We can rename this function and write another function that actually does gridded_inverse_fourier transform that takes in just the stack.

Reply:
done !

Reply:
done